telegram/ext/_handlers/typehandler.py

```diff
@@ -71,13 +74,13 @@ async def callback(update: object, context: CallbackContext)
 
     def __init__(
         self: "TypeHandler[UT, CCT, RT]",
-        type: Type[UT],  # pylint: disable=redefined-builtin
+        type: GenericUT,  # pylint: disable=redefined-builtin
```

Comment:
the type completeness check is not happy with this. Maybe it needs to be `GenericUT[UT]`? 🤔